### ISSUE TITLE
No email notification on tickets

### ISSUE THREAD

**Issue by weeblr** [joomla/jed-issues#117]
**Describe the bug**
When opening a JED ticket, no notification is received in case of new messages or even the ticket being closed.

**To Reproduce**
1. Open a ticket
2. A JED admin posts a message and closes the ticket
3. Nothing happens, have to come back regularly to know if ticket has changed

**Expected behavior**
Receive an email notification when the ticket has some changes.

**Additional context**
This has happened for sh404SEF and wbAMP in my case.

**Comment by anibalsanchez** [comment]
Hi,

If an email has produced bounces in the past, the mail service doesn't try to send the emails again.

Please, confirm if the assigned email is still valid, or update your profile to use a different email. If you are sure that the email is working right now, please, send us a ticket to remove it from the **Bounces List**.

Best Regards

**Comment by weeblr** [comment]
HI Anibal,

There never was a bounce on this email. It's the same account used since sh404SEF was started in 2006 under Mambo, a GMail account which is pretty reliable (and not full) I believe.

Please remove it from the Bounces List and please confirm it is on the Bounces List iin the first place. Again, this email was never interrupted or moved from to/from another provider (it's a gmail account!).

Best regards

**Comment by anibalsanchez** [comment]
I've just answered the ticket. The emails are going out and are received.

Please, send us a new ticket if the issue persists.

**Comment by weeblr** [comment]
Hi Anibal

Thanks for fixing the issue. Was the email on the bounces list? or something else. I opened another ticket as you closed the 1st one before I could reply.

It is important to understand if this can happen again. This address never bounced, unless GMail had a technical issue - which is possible. So maybe the "bounce" process is too harsh and putting an address on the bounce list should only happen after a series of bounce and depending on the nature of the bounce (ie fatal error, temp error, etc)?

Best regards

**Comment by anibalsanchez** [comment]
I did nothing. I just replied to your ticket and confirmed that the email went out.

**Comment by weeblr** [comment]
Hi

Well someone or something did something at some point. I got the first email from no-reply@extensions.joomla.org this morning. The last email from that address was from may 2018 - and that was a newsletter, not a notification. My last notification was from July 23, **2015**.
Just for 2019 I opened 4 tickets, none of them got me any notification until the first one this morning.
I'm glad it's sending now but it sounds a bit like there's an unidentified underlying issue going on.

Best regards
